openpype/modules/kitsu/plugins/publish/integrate_kitsu_note.py

```diff
@@ -11,6 +11,7 @@ class IntegrateKitsuNote(pyblish.api.ContextPlugin):
     families = ["render", "kitsu"]
     set_status_note = False
     note_status_shortname = "wfa"
+    status_exceptions = list()
```

Comment:
I agree with https://github.com/ynput/OpenPype/pull/4537#pullrequestreview-1343248632 it should be conditions -> `status_conditions`. Exceptions are not intuitive and is confusing for coders.

Suggestion: Values of `status_exceptions` does not allow default behavior -> e.g. if `status_exceptions` is empty list. I would suggest to change it to profiles with filters and output value (`True` or `False`).

Reply:
In other words it looks like they must be set, and must be set for every possible short name which may be annoying if you want to change status all the time.

Reply:
I'm working on turning `exceptions` into `conditions` as requested. I'm not sure I understand very well your suggestion, this only meant to add conditions if you need them (For instance, if an asset has been validated on kitsu by the real or a lead, then don't change status), otherwise the list can be left empty.

Reply:
@iLLiCiTiT, IMO the current implementation works the way you expect it to.

Reply:
It was suggestion, not requirement :)